galley/pkg/config/schema/schema.go

```diff
@@ -79,6 +81,28 @@ func (m *Metadata) DirectTransform() *DirectTransform {
 	panic("Metadata.DirectTransform: DirectTransform not found")
 }
 
+// SnapshottedCollectionNames returns an aggregate list of names of collections that are snapshotted.
+func (m *Metadata) SnapshottedCollectionNames() []string {
```

Comment:
What does "snapshotted" mean? All of the collections that currently reside in a snapshot? Or all that could potentially be in a snapshot?

Reply:
This is metadata, so it describes collections that will be contained in a snapshot.